### PR TITLE
Re-enable MongoDB

### PR DESCRIPTION
**Pull request by dpetrick**
Makes MongoDB connector available again in releases.

### CODE REVIEW

File: migration-engine/core/Cargo.toml [hunk 1]
```diff
@@ -25,6 +25,6 @@ url = "2.1.1"
 tokio = { version = "1.0", default-features = false, features = ["macros"] }
 
 [features]
-default = ["sql"]
+default = ["sql", "mongodb"]
```
Comment (tomhoule) [inline migration-engine/core/Cargo.toml:28]: nitpick: do we need it the ME for releases? My outsider understanding is that people are only interested about testing mongo with the client, for now

**Review by tomhoule — APPROVED** [review]
Left a question comment, in general LGTM!

File: migration-engine/core/Cargo.toml (hunk 1 above)
Comment (dpetrick) [reply migration-engine/core/Cargo.toml:28]: Maybe, maybe not. It has minimal tooling to create and drop things. If it's a problem for the ME team that the dependency is added due to size, we can probably drop it.

Comment (tomhoule) [reply migration-engine/core/Cargo.toml:28]: I guess we can just add it, and remove it if it causes any problem

**Pull request merged**